docs/core/development/policy/coding-standard.rst

```diff
@@ -640,7 +639,7 @@ All SQL keywords should be in upper case.
 C Code
 ------
 
-C code must be optional, and work across multiple platforms (MSVC++9/10/14 for Pythons 2.7 and 3.5+ on Windows, as well as recent GCCs and Clangs for Linux, macOS, and FreeBSD).
+C code must be optional, and work across multiple platforms (MSVC++9/10/14 for 3.6+ on Windows, as well as recent GCCs and Clangs for Linux, macOS, and FreeBSD).
```

Comment:
I have no idea which MSVC++ version is required ...I remember that the C++ compiler has now a different name.. .like Windows 10 SDK or something like this.

But I think that the C code poilicy is now to move any C code outside of this repo :)

```suggestion
C code must be optional, and work across multiple platforms (MSVC++9/10/14 for Python 3 on Windows, as well as recent GCCs and Clangs for Linux, macOS, and FreeBSD).

It should be distributed as a separate package and not  as part of the main Twisted project.
```

Reply:
this policy is already here:
https://github.com/twisted/twisted/pull/1528/files/92cd5ad286fc947645867667f218b12eeb9459ec#diff-514cad77343003f66671092af193f29ac569e2a6b6bd04ffd62ff468f17b6f07R644

Reply:
```suggestion
C code must be optional, and work across multiple platforms (MSVC++14 for Python 3 on Windows, as well as recent GCCs and Clangs for Linux, macOS, and FreeBSD).
```

https://wiki.python.org/moin/WindowsCompilers#Which_Microsoft_Visual_C.2B-.2B-_compiler_to_use_with_a_specific_Python_version_.3F